.github/workflows/instrumented-test-recording.sh

```
@@ -0,0 +1,22 @@
+
+API_LEVEL=$1
+ORIENTATION=$2
+
+set -x
+set +e
+echo "Starting instrumented tests..."
+./gradlew connectedCheck &
+TEST_PID=$!
+sleep 10
+echo "Starting the screen recording..."
+adb exec-out "while true; do screenrecord --bugreport --output-format=h264 -; done" | ffmpeg -i - testRecording-$API_LEVEL-$ORIENTATION.mp4 &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)